spring-web/src/test/java/org/springframework/web/filter/RequestLoggingFilterTests.java

```diff
@@ -44,8 +44,96 @@
  */
 public class RequestLoggingFilterTests {
 
-	private final MyRequestLoggingFilter filter = new MyRequestLoggingFilter();
+	private MyRequestLoggingFilter filter = new MyRequestLoggingFilter();
 
+	@BeforeEach
+	public void reset() {
+		filter = new MyRequestLoggingFilter();
+	}
+
+	@Test
+	public void defaultPrefix() throws Exception {
+		final MockHttpServletRequest request = new MockHttpServletRequest("POST", "/hotels");
+		MockHttpServletResponse response = new MockHttpServletResponse();
+
+		FilterChain filterChain = new NoOpFilterChain();
+		filter.doFilter(request, response, filterChain);
+
+		assertThat(filter.beforeRequestMessage).isNotNull();
+		assertThat(filter.beforeRequestMessage.startsWith(AbstractRequestLoggingFilter.DEFAULT_BEFORE_MESSAGE_PREFIX)).isTrue();
```

Comment:
```suggestion
		assertThat(filter.beforeRequestMessage).startsWith(AbstractRequestLoggingFilter.DEFAULT_BEFORE_MESSAGE_PREFIX);
```

With AssertJ, there's no need to check for "not null" and "is true". Simply use `.startsWith(...)` which handles both of those use cases.

Please apply the above two changes to all affected test methods.